packages/flutter/lib/src/widgets/expansible.dart

```diff
@@ -340,9 +349,34 @@ class _ExpansibleState extends State<Expansible> with SingleTickerProviderStateM
     widget.controller.removeListener(_toggleExpansion);
     _animationController.dispose();
     _heightFactor.dispose();
+    _timer?.cancel();
+    _timer = null;
     super.dispose();
   }
 
+  void _announceSemantics() {
+    if (!MediaQuery.supportsAnnounceOf(context)) {
+      return;
+    }
+    final WidgetsLocalizations localizations = WidgetsLocalizations.of(context);
+    final String stateHint = widget.controller.isExpanded
```

Comment:
Consider pass-in the boolean or message directly. or refactor out something like
```
bool get _effectiveExpanded => widget.controller.isExpanded;
```
in case we change how we decide _effectiveExpanded later